packages/core/src/lib/markdown-it/markdown-it-dimmed.js

```diff
@@ -1,125 +0,0 @@
-'use strict';
```

Comment:
one minor nit: let's find out where this was from and add in the license notice, if there's one!

Reply:
I think the code was adapted using the below as reference:
https://github.com/markdown-it/markdown-it/blob/master/lib/rules_inline/strikethrough.js

The license of the repository can be found here:
https://github.com/markdown-it/markdown-it/blob/master/LICENSE

In fact, we have already written similar licensing notice before in `markdown-it-footnotes.js` so I'm going to copy the template over :-)